packages/flutter_tools/lib/src/commands/emulators.dart

```diff
@@ -70,17 +77,27 @@ class EmulatorsCommand extends FlutterCommand {
     }
   }
 
+  Future<Null> _createEmulator(String name) async {
```

Comment:
Shouldn't `name` be optional here?

Reply:
Cool, I think it just makes it a bit more explicit that it can be null (in the absence of non-null annotations).